core/tests/puppeteer-acceptance-tests/utilities/user/curriculum-admin.ts

```diff
@@ -807,6 +682,81 @@ export class CurriculumAdmin extends BaseUser {
   async openExplorationControlDropdown(): Promise<void> {
     await this.clickOn(explorationControlsSettingsDropdown);
   }
+
+  /**
+   * Function to unpublish a topic.
+   * @param {string} topicName - The name of the topic to unpublish.
+   */
+  async unpublishTopic(topicName: string): Promise<void> {
+    await this.openTopicEditor(topicName);
+    await this.clickOn(' Unpublish Topic ');
+  }
+
+  /**
+   * Function to delete a topic.
+   * @param {string} topicName - The name of the topic to delete.
+   */
+  async deleteTopic(topicName: string): Promise<void> {
+    await this.goto(testConstants.URLs.TopicAndSkillsDashboard);
+    await this.clickOn('.e2e-test-topic-edit-box');
```

Comment:
Please set them as constants at the beginning of this file, and also for the ones below.

Reply:
done.